src/bsoncxx/CMakeLists.txt

```diff
@@ -158,10 +158,7 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
     DESTINATION ${BSONCXX_HEADER_INSTALL_DIR}
     COMPONENT dev
     FILES_MATCHING
-      PATTERN "*.h"
```

Comment:
Are you sure we don't want to export any C headers?  Haven't looked, but thought I'd ask.